### PR TITLE
[DOCS] Add docs about deploy platforms

### PR DESCRIPTION
**Pull request by arti0090**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.10 |
| Bug fix?        | no                                                      |
| New feature?    | no                                                      |
| BC breaks?      | no/yes                                                       |
| License         | MIT                                                          |

Created unordered list of files as in tip it otherwise does not break lines between links
image

### CODE REVIEW

**Review by CoderMaggie — APPROVED** [review]

File: docs/getting-started-with-sylius/deployment.rst [hunk 1]
```diff
@@ -9,4 +9,8 @@ Check it out!
 
 .. tip::
 
-    👉 :doc:`How to deploy Sylius to Platform.sh? </cookbook/deployment/platform-sh>`
+    - 👉 :doc:`How to deploy Sylius to Platform.sh? </cookbook/deployment/platform-sh>`
+    - 👉 `How to deploy on Platform.sh? <https://docs.platform.sh>`_
```
Comment (CoderMaggie) [inline docs/getting-started-with-sylius/deployment.rst:13]:
```suggestion
    - 👉 `How to deploy on Platform.sh? <https://docs.platform.sh>`_
```

**Review by CoderMaggie — CHANGES_REQUESTED** [review]

File: docs/getting-started-with-sylius/deployment.rst [hunk 2]
```diff
@@ -9,4 +9,8 @@ Check it out!
...
+    - 👉 `How to deploy on Symfony Cloud? <https://symfony.com/cloud/>`_
+    - 👉 `How to deploy on Artifakt.com? <https://docs.artifakt.com/>`_
+    - 👉 `How to deploy on Cloudways PHP Hosting? <https://support.cloudways.com/en/>`_
```
Comment (CoderMaggie) [inline docs/getting-started-with-sylius/deployment.rst:16]:
```suggestion
    - 👉 :doc:`</cookbook/deployment/platform-sh>`
    - 👉 :doc:`</cookbook/deployment/cloudways>`
    - 👉 :doc:`</cookbook/deployment/symfonycloud>`
    - 👉 :doc:`</cookbook/deployment/artifakt>`
```

**Review by CoderMaggie — CHANGES_REQUESTED** [review]

**Comment by arti0090** [comment]
🎉 
image

**Review by CoderMaggie — APPROVED** [review]

File: docs/getting-started-with-sylius/deployment.rst [hunk 3]
```diff
@@ -9,4 +9,15 @@ Check it out!
...
+    - 👉 :doc:`How to deploy Sylius to Artifakt? </cookbook/deployment/symfonycloud>`
+    - 👉 :doc:`How to deploy Sylius to Cloudways? </cookbook/deployment/artifakt>`
+
+Learn more about the deployment platforms
+-----------------------------------------
+
+    - `Platform.sh <https://docs.platform.sh>`_
+    - `Symfony Cloud <https://symfony.com/cloud/>`_
+    - `Artifakt.com <https://docs.artifakt.com/>`_
+    - `Cloudways PHP Hosting <https://support.cloudways.com/en/>`_
```
Comment (CoderMaggie) [inline docs/getting-started-with-sylius/deployment.rst:23]:
```suggestion
- `Platform.sh <https://docs.platform.sh>`_
- `Symfony Cloud <https://symfony.com/cloud/>`_
- `Artifakt.com <https://docs.artifakt.com/>`_
- `Cloudways PHP Hosting <https://support.cloudways.com/en/>`_
```

**Review by lchrusciel — APPROVED** [review]

**Pull request merged**